kadi/commands/states.py

```diff
@@ -711,7 +714,7 @@ class SPMEnableTransition(FixedTransition):
 
     command_attributes = {"tlmsid": "AOFUNCEN"}
     command_params = {"aopcadse": 30}
-    state_keys = ["sun_pos_mon"]
+    state_keys = SPM_STATE_KEYS
```

Comment:
This speaks to a lack of knowledge on my part, but for a non-eclipse transition, are the battery_connect and eclipse_enable_spm keys relevant?

Reply:
This tells the code that if you are asking for (e.g.) `battery_connect` that it should include this class in the processing that looks for potential state-changing commands since `battery_connect` and `sun_pos_mon` are coupled in some way. I'm honestly not 100% sure if this is needed but it's generally a good idea for states that work in tandem.

Reply:
OK, that's fine.  It seemed not necessary in passing but also seems like it would be benign if not really needed (and is tested this way as fine).